yb-voyager/cmd/exportDataDebezium.go

```diff
@@ -403,10 +410,29 @@ func reportStreamingProgress() {
 	}
 }
 
+func updateCallhomeEventsStats() {
+	//In case the pb is disabled, this will update the stats in 15 minutes
```

Comment:
just check: the comment says 15 mins, but i sleep of 12 mins

Reply:
its just a miss in updating comment, as I changed the sleep as per review of earlier PRs.